### PR TITLE
Added note w.r.t On-Premises mailbox's experience

### PR DESCRIPTION
**Pull request by Batre-MSFT**
Added note to clarify what will be On-Premises mailbox experience be after steps in this article are performed

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeHybrid/hybrid-deployment/set-up-microsoft-365-groups.md [hunk 1]
```diff
@@ -26,7 +26,8 @@ Learn how to enable on-premises Exchange users to use Microsoft 365 Groups in a
...
 
 > [!IMPORTANT]
-> Using Microsoft 365 Groups with on-premises users in an Exchange hybrid deployment is a new feature. Because it's so new, you might run into some issues when you set it up. Check the [Known issues](#known-issues) section at the end of this topic for fixes to known issues.
+After the steps of this article are successfully performed, the On-Premises mailbox can send and receive emails to Microsoft 365 Group. However, the Microsoft 365 Group will not appear in the navigation bar of Outlook on the web or Outlook desktop client. For full fledged Microsoft 365 Group exprience, the mailbox must be present in Microsoft 365.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeHybrid/hybrid-deployment/set-up-microsoft-365-groups.md:29]:
```suggestion
After the steps of this article are successfully performed, the On-Premises mailbox can send and receive emails to and from a Microsoft 365 Group. However, the Microsoft 365 Group will not appear in the navigation bar of Outlook on the web or Outlook desktop client. For a full-fledged Microsoft 365 Group experience, the mailbox must be present in Microsoft 365. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

File: Exchange/ExchangeHybrid/hybrid-deployment/set-up-microsoft-365-groups.md [hunk 2]
```diff
@@ -26,7 +26,8 @@ Learn how to enable on-premises Exchange users to use Microsoft 365 Groups in a
...
 
 > [!IMPORTANT]
-> Using Microsoft 365 Groups with on-premises users in an Exchange hybrid deployment is a new feature. Because it's so new, you might run into some issues when you set it up. Check the [Known issues](#known-issues) section at the end of this topic for fixes to known issues.
+After the steps of this article are successfully performed, the on-premises mailbox can send and receive emails to and from a Microsoft 365 group. However, the Microsoft 365 group will not appear in the navigation bar of Outlook on the web or Outlook desktop client. For a fully-fledged Microsoft 365 Groups experience, the mailbox must be present in Microsoft 365.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeHybrid/hybrid-deployment/set-up-microsoft-365-groups.md:29]:
```suggestion
After the steps in this article are successfully performed, the on-premises mailbox can send and receive emails to and from a Microsoft 365 group. However, the Microsoft 365 group will not appear in the navigation bar of Outlook on the web or Outlook desktop client. For a fully-fledged Microsoft 365 Groups experience, the mailbox must be present in Microsoft 365. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @Batre-MSFT - one more minor change, please.

**Pull request merged**